package.json

```diff
@@ -29,26 +29,27 @@
   },
   "homepage": "https://github.com/wikimedia/service-template-node",
   "dependencies": {
-    "bluebird": "^3.5.0",
-    "body-parser": "^1.17.1",
-    "bunyan": "^1.8.9",
+    "bluebird": "^3.5.1",
+    "body-parser": "^1.18.2",
+    "bunyan": "^1.8.12",
     "cassandra-uuid": "^0.0.2",
-    "compression": "^1.6.2",
-    "domino": "^1.0.28",
-    "express": "^4.15.2",
-    "js-yaml": "^3.8.2",
+    "compression": "^1.7.1",
+    "domino": "^1.0.30",
+    "express": "^4.16.2",
+    "js-yaml": "^3.10.0",
     "preq": "^0.5.2",
-    "service-runner": "^2.2.5",
-    "swagger-router": "^0.5.6",
-    "swagger-ui": "git+https://github.com/wikimedia/swagger-ui#master"
+    "service-runner": "^2.4.0",
```

Comment:
Let's bump it to 2.4.2. Also, let's bump preq to 0.5.3 once wikimedia/preq#30 is in.